widget/text_test.go

```diff
@@ -706,3 +706,106 @@ func TestText_lineBounds_variable_char_width(t *testing.T) {
 		})
 	}
 }
+
+func TestText_binarySearch(t *testing.T) {
+	tests := []struct {
+		name string
+		text string
+		want int
+	}{
+		{
+			name: "IM_Test",
```

Comment:
The result for this test will be reported as "TestText_binarySearch_IM_Test" - the "_Test" suffix is unnecessary